src/layers/core/icon-layer/icon-layer.js

```diff
@@ -60,7 +60,8 @@ const defaultProps = {
   getPosition: x => x.position,
   getIcon: x => x.icon,
   getColor: x => x.color || DEFAULT_COLOR,
-  getSize: x => x.size || 1
+  getSize: x => x.size || 1,
+  getAngle: x => x.angle || 0
```

Comment:
> Looks like it expects angles to be in radian. I personally think degrees is more user friendly 

- Unfortunately, regardless of what we choose, I think this will always trip people up, unless we call this `angleRadians` or similar. While ugly, it could be the right choice.
- Whatever we choose we need to be consistent across our layers - this needs to go into API design guidelines doc.